conceptual/Npgsql/types/basic.md

```diff
@@ -66,7 +66,7 @@ multirange types (PG14)     | NpgsqlRange\<TElement>[]   |
 enum types                  | TEnum                      |
 array types                 | Array (of element type)    |
 
-<sup>1</sup>Types which can be read as `byte[]` can also be written as `ReadOnlyMemory<byte>` and `Stream`., and if they could be big, consider using `Stream`CommandBehavior.SequentialAccess` and `NpgsqlDataReader.GetStream()` to stream the row and column instead of buffering them. Note that if you're reading string data (e.g. `jsonb`) as binary, it's you're responsibility to perform UTF8 decoding (or whatever encoding is configured for your database).
+<sup>1</sup>Types which can be read as `byte[]` can also be written as `ReadOnlyMemory<byte>` and `Stream`., and if they could be big, consider using `StreamCommandBehavior.SequentialAccess` and `NpgsqlDataReader.GetStream()` to stream the row and column instead of buffering them. Note that if you're reading string data (e.g. `jsonb`) as binary, it's you're responsibility to perform UTF8 decoding (or whatever encoding is configured for your database).
```

Comment:
It probably should have been just `CommandBehavior.SequentialAccess` and `Stream` there is wrong. Also, there seems to be an unnecessary `.` after the first `Stream`.

Reply:
And another, in the last sentence `you're` should have been `your`.